### ISSUE TITLE
Loss is different from that in paper?

### ISSUE THREAD

**Issue by YeDeming** [neulab/RIPPLe#5]
Hi,

Thanks for releasing the nice source code. 

As 

https://github.com/neulab/RIPPLe/blob/e1d20fb96afa5a18f18617daa4427d55d0c36e26/constrained_poison.py#L376 

shows,
loss = ref_loss + args.L * inner_prod,

but the ref_loss belongs to the clean data, and std_loss belongs to '(L_P) in the paper'.

In equation (4) of the paper, you optimize  L_P + the dot-products. But in the code, you optimize the L_FT + the dot-products?

Looking forward to you reply. Thanks!

Best,
Deming

**Comment by pmichel31415** [comment]
Hi @YeDeming , thanks for pointing this out. This is my error when commenting Keita's code. Looking at https://github.com/neulab/RIPPLe/blob/paul_refactor/poison.py#L912 it looks like ref_* refers to the poisoning data rather than the other way around

**Comment by YeDeming** [comment]
ok, thanks!
